doc/model/axes/axis-cartesian-continuous.xml

```diff
@@ -360,9 +375,70 @@
                 Elements that are drawn beyond its area are clipped.
 
                 This property takes precedence over the property <tt>originIsZero</tt>.
+
+
+
+                This property is ignored if <tt>fixedMin</tt> and <tt>fixedLength</tt> are both specified. The behavior is described as follows:
+                <!-- NEW603 -->
+                <ul>
+                    <li>  <tt>fixedMax</tt>, <tt>fixedLength</tt> specified, without <tt>fixedMin</tt>: a fixed minimum is implied from the max and length values </li>
+                    <li>  <tt>fixedMin</tt>, <tt>fixedMax</tt> specified, without <tt>fixedLength</tt>: the fixed maximum and minimum are set directly</li>
+                    <li>   All options specified: <tt>fixedMax</tt> is ignored and the behavior corresponds to the first case
+                    </li>
+                </ul>
+
+                See related axis options:
+                <c:link to="#fixedMin" />,
+                <c:link to="#fixedLength" />
+
             </c:documentation>
         </c:property>
 
+        <c:property name="fixedLength" type="number pvc.options.varia.TimeInterval" category="Continuous > Scale">
+            <c:documentation>
+                <!-- NEW603 -->
+                The fixed length of the domain that the axis will show (applies to numeric axes).
+                
+                A string value is converted to a number.
+                
+                The value may be bigger or smaller than the actual
+                length of the data.
+                
+                This property takes precedence over the property <tt>originIsZero</tt>, if specified either with 
+                <tt>fixedMin</tt>, <tt>fixedMax</tt> or <tt>DomainAlign</tt>.
+
+                This property can be specified with several options to define the initial domain, 
+                that take the following precedence order : 
+
+                <ul>
+                    <li> <tt>fixedMin</tt> - The fixed minimum is set and a fixed maximum is calculated, so that the length shown corresponds to the defined fixed length </li>
+                    <li> <tt>fixedMax</tt> - The fixed maximum is set and a fixed minimum is calculated, so that the length shown corresponds to the defined fixed length </li>
+                    <li> <tt>DomainAlign</tt> - Fixed minimums and maximums are set so that the length shown corresponds to the defined fixed length; depending on the domain
```

Comment:
I found the `domainAlign` description confusing. There's _one_ implied fixed limit, depending on the chosen alignment. The other limit is not fixed but "floating" and aligns to data. Maybe it's easier to describe each of the cases?